internal/admin/admin.go

```diff
@@ -716,6 +720,26 @@ func (adm *Admin) updateLogStream(ctx context.Context, lsid types.LogStreamID, p
 	return newLSDesc, nil
 }
 
+func (adm *Admin) hasSealedReplica(ctx context.Context, lsdesc *varlogpb.LogStreamDescriptor) bool {
+	for _, replica := range lsdesc.Replicas {
+		meta, err := adm.snmgr.GetMetadata(ctx, replica.StorageNodeID)
+		if err != nil {
+			continue
+		}
```

Comment:
If a replica behaves badly or slowly, hasSealedReplica seems to return a false negative.

Reply:
Right! If GetMetadata fails and it is not clear whether it is sealed, updateLS is not performed.

Reply:
Aha, I got your intention. It's okay to keep the log stream. 😃